utils/factomUtils.js

```diff
@@ -0,0 +1,19 @@
+const DidCreationStatus = Object.freeze({
+    FINISHED: "finished",
+    PENDING: "pending",
+    NOT_FOUND: "not found",
+});
+
+const didStatusFromEntryAck = ({ entrydata: entryData }) => {
+    if (entryData.status === 'DBlockConfirmed') {
+        return DidCreationStatus.FINISHED;
+    }
+    if (entryData.status === 'NotConfirmed' || entryData.status === 'TransactionACK') {
```

Comment:
To treat NotConfirmed as Pending is a bit of a risk. It the local node has a problem or is restarted the entry will not hit the network. I think we can just accept that

Reply:
Yeah, as this is for user feedback, I'm not sure what the user could do in such a situation. My understanding was that it will end up in a "not found" eventually and the user will have to try again.

Reply:
Yeah, you would need some background task to check it to ensure eventual registration,  but that is a bit too much for this project